src/app/lib/config/services/scotland.js

```diff
@@ -22,7 +22,7 @@ const service = {
     brandName: 'BBC Scotland',
     product: 'BBC Scotland',
     defaultImage:
-      'https://www.bbc.co.uk/news/special/2015/newsspec_10857/bbc_news_logo.png',
+      'https://news.files.bbci.co.uk/include/articles/public/scotland/images/og/poster-1024x576.png',
```

Comment:
Am I right in thinking that when this gets merge this URL will resolve itself?

Reply:
> Am I right in thinking that when this gets merged this URL will resolve itself?
That was my thinking...